Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.13.0-slim
+ENV PYTHONUNBUFFERED 1
+WORKDIR app/
```

Comment:
The `WORKDIR` should be an absolute path. Consider changing it to `/app` instead of `app/`.